src/template-asciidoctor-umd.js

```diff
@@ -3,43 +3,7 @@ if (typeof Opal === 'undefined' && typeof module === 'object' && module.exports)
 }
 
 if (typeof Opal === 'undefined') {
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#Fundamental_objects
-  var fundamentalObjects = [
```

Comment:
Opal does not touch fundamental objects anymore :tada: :tada: :tada:

Reply:
:tada: A day we had only dreamed of before!